Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -271,8 +288,32 @@ public InstallerVM(ILogger<InstallerVM> logger, DTOSerializer dtos, SettingsMana
                         .Where(t => t.Failed)
                         .Concat(Validate())
                         .ToArray();
-                    if (!errors.Any()) return ErrorResponse.Success;
-                    return ErrorResponse.Fail(string.Join("\n", errors.Select(e => e.Reason)));
+                    if (errors.Length == 0)
```

Comment:
Might be me, but I find the logic here pretty tricky to follow with the UnrecognisedFilesPresent bool. I'll have a look to see if I can lower the code complexity a bit here

Reply:
its a casualty of working around the system that was really designed to return a "error or not error" binary paradigm to block activation of the start button.

I Didnt want to and couldnt even if I did want to, rewrite the entire validate() system , so just tacked on the special case of Unrecognised files being present as being distinct from recognised files being present. 

to fit in the logic for when to enable the start button based on two boolean branches

```
 this.WhenAnyValue(x => x.ViewModel.ErrorState, x => x.ViewModel.IsShiftKeyPressed, x => x.ViewModel.UnrecognisedFilesPresent)
    .Select(v => (!v.Item1.Failed || v.Item1.Succeeded ) || (v.Item1.Failed && v.Item2 && v.Item3))
```

So that the shift key is only viable as a shortcut in that one error scenario.

Its ugly but yeah, im sure theres  a better way.